orttraining/orttraining/training_api/checkpoint.cc

```diff
@@ -169,12 +181,15 @@ Status ToFile(const PathString& checkpoint_path, flatbuffers::FlatBufferBuilder&
  * @param trainable_tensor_protos trainable parameters in TensorProto format.
  * @param non_trainable_tensor_protos non-trainable parameters in TensorProto format.
  * @param checkpoint_path file where checkpoint is saved.
+ * @param nominal_checkpoint if true, create a nominal checkpoint.
+ * @param external_data_threshold optional threshold in bytes for external data. If the size of the data of the
+ *                                TensorProtos exceeds this threshold, then we save the data in an external data file.
  * @return Status
  */
-Status FromTensorProtos(
-    gsl::span<const ONNX_NAMESPACE::TensorProto> trainable_tensor_protos,
-    gsl::span<const ONNX_NAMESPACE::TensorProto> non_trainable_tensor_protos,
-    const PathString& checkpoint_path, const bool nominal_checkpoint) {
+Status FromTensorProtos(gsl::span<const ONNX_NAMESPACE::TensorProto> trainable_tensor_protos,
+                        gsl::span<const ONNX_NAMESPACE::TensorProto> non_trainable_tensor_protos,
+                        const PathString& checkpoint_path, const bool nominal_checkpoint,
+                        const int32_t external_data_threshold = 1800) {
```

Comment:
if we're going to use the same default value for external_data_threshold in multiple places, make it a const.

Do we need to have a separate default here though or can the value from SaveCheckpoint be passed through?